x-pack/plugin/kql/src/test/java/org/elasticsearch/xpack/kql/query/KqlQueryBuilderTests.java

```diff
@@ -34,6 +36,10 @@
 import static org.hamcrest.Matchers.nullValue;
 
 public class KqlQueryBuilderTests extends AbstractQueryTestCase<KqlQueryBuilder> {
+    @Before
```

Comment:
I think the following works as well, so it avoids skipping the individual tests?

```suggestion
    @BeforeClass
```

Reply:
Done!